recipes/nspr/all/conanfile.py

```diff
@@ -1,12 +1,14 @@
 from conan.tools.microsoft import msvc_runtime_flag
-from conan.tools.files import rename
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+from conan.tools.files import rename, get, rmdir, chdir, replace_in_file
+from conan.tools.scm import Version
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conans import tools, AutoToolsBuildEnvironment
 import contextlib
 import functools
 import os
 
-required_conan_version = ">=1.36.0"
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
Version() does not use .major or .minor here, so 1.47.0 is enough